google-cloud-bigtable/pom.xml

```diff
@@ -341,6 +341,7 @@
       <id>enable-verbose-grpc-logs</id>
       <properties>
         <bigtable.enable-grpc-logs>true</bigtable.enable-grpc-logs>
+        <java.util.logging.config.file>src/test/resources/logging-verbose.properties</java.util.logging.config.file>
```

Comment:
Maven is very confusing about this. There are 2 sets of properties - 
1. maven properties
2. java system properties
This set of properties are maven properties. So this doesnt quite work.

You need to:
1. In the top level properties, define a new maven property:
   `<java-log-config>src/test/resources/logging.properties</java-log-config>`
2. in the `enable-verbose-grpc-logs` profile add a property to override the log config:
     `<java-log-config>src/test/resources/logging.properties</java-log-config>`
3. in the failsafe plugin config use the variable:
```
    <configuration>
         ....
          <systemPropertyVariables>
          <java.util.logging.config.file>${java-log-config}</java.util.logging.config.file>
        ...
</configuration>
```

Reply:
done